src/main/dynamo/Beyonce.ts

```diff
@@ -186,6 +188,52 @@ export class Beyonce {
       .promise()
   }
 
+  /** Performs a (partial) update on an existing item in Dynamo.
+   *  Partial updates do not work with JayZ encryption and this method
```

Comment:
Context on why this doesn't work with JayZ encryption:

Currently JayZ uses one encryption key for the entire item. And that key is stored (encrypted by KMS) in a  `__jayz__metadata` field on that item. So when you want to update a field, you have to encrypt it, but you can't just replace the item's encrypted key because all the other fields use it too. 

We could get around this by using 1 key per field. But that would balloon item size to the point where it doesn't seem worth it. In the context of JayZ encryption, you might as well just read the full item, and overwrite it.